GNUmakefile

```diff
@@ -132,3 +133,7 @@ generate-doc: ## Generate the resource documentation via tfplugindocs
 .PHONY: update-tf-compatibility-matrix
 update-tf-compatibility-matrix: ## Update Terraform Compatibility Matrix documentation
 	./scripts/update-tf-compatibility-matrix.sh
+
+.PHONY: generate-changelog
+generate-changelog: ## Updates Unreleased section of changelog file
```

Comment:
knitpit: could we find a name that wouldn't need a comment/explanation?   like generate-changelog-unreleased-section ?

Reply:
updated to make name more explicit 👍